std/net/curl.d

```diff
@@ -235,8 +232,11 @@ version(unittest)
         immutable(T)[] bdy;
     }
 
-    private Request!T recvReq(T=char)(Socket s)
+    private Request!T recvReq(T=char, S)(S s)
     {
+        import std.socket : Socket;
+        static assert (is(S == Socket), "Invalid type used. Use std.socket.Socket.");
```

Comment:
I really wanted to avoid that there are different imports available during the unittest mode, hence this workaround.

Reply:
Ew.

I think this cure is worse than the disease.